src/services/rateSource/NBUService.ts

```diff
@@ -0,0 +1,38 @@
+import { RateSourceResponse, RateSourceService } from './RateSourceService';
+import responseMessages from '../../constants/responseMessages';
+
+type NBUResponse = {
```

Comment:
isn't it more like an interface rather then type?

Reply:
In my case it's an array
So it's a type
I know I could declare interface and then add brackets, I just like it this way ¯\_(⊙︿⊙)_/¯